HTTP/Request2.php

```diff
@@ -926,11 +926,6 @@ public function send()
         if (empty($this->adapter)) {
             $this->setAdapter($this->getConfig('adapter'));
         }
-        // magic_quotes_runtime may break file uploads and chunked response
```

Comment:
HTTP_Request2 currently works on PHP 5.2+, magic_quotes_runtime is disabled from PHP 5.4 onwards. This will break 5.2 and 5.3 support on installations having magic_quotes enabled (unlikely, I know, but still).

So we need to either up the requirements to PHP 5.4 or drop this PR.

Reply:
Considering how (hopefully) unlikely it is that people are still using even PHP5.6...perhaps the best thing to do is up the requirements to PHP5.4?

Reply:
What are you trying to resolve here, anyway? get_magic_quotes_runtime() is neither deprecated nor removed and set_magic_quotes_runtime() can only be called on 5.2 or 5.3 with magic_quotes_runtime enabled.

Reply:
Well, looking on travis build results below, maybe it's easier to drop support for 5.2 and 5.3 anyway. It is increasingly difficult to test on these.